engine/src/sequence.cpp

```diff
@@ -169,7 +169,11 @@ bool Sequence::loadXML(QXmlStreamReader &root)
     if (scene != NULL)
     {
         sceneValues = scene->values();
+#if (QT_VERSION < QT_VERSION_CHECK(5, 13, 0))
         qSort(sceneValues.begin(), sceneValues.end());
+#else
+	(sceneValues.begin(), sceneValues.end());
```

Comment:
this won't even build

Reply:
sorry, my fault. i forgot the function call...